tools-v2/pkg/cli/command/curvefs/list/mountpoint/mountpoint.go

```diff
@@ -100,6 +100,13 @@ func updateTable(table *table.Table, info *mds.ListClusterFsInfoResponse) {
 	fssInfo := info.GetFsInfo()
 	rows := make([]map[string]string, 0)
 	for _, fsInfo := range fssInfo {
+		if len(fsInfo.GetMountpoints()) == 0 {
+			row := make(map[string]string)
+			row["fs id"] = fmt.Sprintf("%d", fsInfo.GetFsId())
```

Comment:
maybe you can use const string for row filed.

Reply:
> maybe you can use const string for row filed.

After this, I will submit a pr to modify it. There is also this problem later pr.